pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -165,6 +165,25 @@ func (c *cephStatusChecker) updateCephStatus(status *cephclient.CephStatus, cond
 	// Update with Ceph Status
 	cephCluster.Status.CephStatus = toCustomResourceStatus(cephCluster.Status, status)
 
+	// versions store the ceph version of all the ceph daemons and overall cluster version
+	versions, err := cephclient.GetAllCephDaemonVersions(c.context, c.clusterInfo)
+	if err != nil {
+		logger.Errorf("failed to get ceph daemons versions. %v", err)
+		return
```

Comment:
Actually, I don't think we want to return here. Just log and put the assignment in a `else`. We need to update the cluster condition at the end, even if this fails.

Reply:
Done.